lib/error.g

```diff
@@ -1,9 +1,24 @@
 #############################################################################
 ##
-#W  error.g                    GAP library                 Steve Linton
+##  GAP, a computer algebra system
```

Comment:
I'd prefer replacing `computer` by `computational`.

Reply:
Fine by me. But perhaps it should instead be "system for computational discrete algebra "?

On the website, we say:
> GAP - Groups, Algorithms, Programming - a System for Computational Discrete Algebra

We also say this on the website and in the README:
> GAP is a system for computational discrete algebra

The reference manual says:
> GAP is a free, open and extensible software package for computation in discrete abstract algebra

Reply:
any of the three fine w/me, sure.